### PR TITLE
Update adopters.csv

### PR DESCRIPTION
**Pull request by cirino**
# Contributor Covenant Adoption

## Project name
Casa do Dev

## Project repository
https://github.com/casadodev/aprender

## Link to code of conduct in your project's repo or documentation
[Code of conduct](https://github.com/casadodev/aprender/blob/main/code_of_conduct.md)

## Check your work!
Please double-check that you inserted the contact method for reporting incidents in the template. (Search the document for `[https://twitter.com/casadodev]`)

*Thank you!*

### CODE REVIEW

**Review by Nuttapon-Makchoos — COMMENTED** [review]
.

**Review by Nuttapon-Makchoos — COMMENTED** [review]
.

File: static/adopters.csv [hunk 1]
```diff
@@ -397,3 +397,4 @@ Your First PR, https://github.com/yourfirstpr/
 ZoneMinder, https://github.com/ZoneMinder/
 AB3C, https://ab3c.org.br/
 Open Privacy Tech, https://github.com/openprivacytech
+Casa do Dev, https://github.com/casadodev/aprender
```
Comment (GAMEZs9) [inline static/adopters.csv:400]: @GAMEZs9

**Review by Nuttapon-Makchoos — COMMENTED** [review]
.

**Comment by CoralineAda** [comment]
@cirino You still have [INSERIR MÉTODO DE CONTATO] in your document-- this is where you need to put in your contact information for reporting an incident. Thanks!

**Comment by cirino** [comment]
Aww thank you. I wrote in the wrong place and forgot to update.

It has already been added to the document.

**Pull request closed without merging**